base/bashrc.d/900-ws-info

```diff
@@ -0,0 +1,137 @@
+(
+	set -eu
+	function remove_self() {
+		# Remove the hook file, because we no longer need it to be loaded.
+		if [[ "${BASH_SOURCE[0]}" =~ \.bashrc.d/[0-9]+- ]]; then {
+			rm -f "${BASH_SOURCE[0]}"
+		} fi
+	}
+	lockfile=/tmp/.wsinfo_lck
+	trap '{ remove_self; rm -f "$lockfile"; } || :' EXIT ERR
+
+	# Return early if user opted out
+	if test "${GITPOD_DISABLE_WORKSPACE_IMAGE_INFO:-false}" == true; then {
+		exit 0 # This is a subshell
+	} fi
+
+	if test ! -e "$lockfile"; then {
+		remove_self
+		printf '' > "$lockfile"
+
+		# The supervisor creates the task terminals, supervisor calls BASH from `/bin/bash` instead of the realpath `/usr/bin/bash`
+		# So we will only print the info once, on a non-task terminal that the user created manually
```

Comment:
@axonasif as an alternative, it would be nice if this output were visible in a separate (exclusive) task terminal, avoiding user opened terminals altogether. In other words, then, they could always "reference it", by simplying looking at it, without having to ever opt-out.

Another thought is, we persist this output somewhere in `/workspace`, and then link to it from the Gitpod menu "in workspace". In other words, never present this content in a Terminal.

:point_up: are example thoughts that would be best to socialize as feedback in a RFC, as an example. :pray:

Reply:
I think it is good idea to have one task terminal which shows such info. I also like idea of doing it in supervisor only for regular workspaces.